packages/@aws-cdk-testing/framework-integ/test/aws-codedeploy/test/server/integ.deployment-group.ts

```diff
@@ -40,6 +41,9 @@ new codedeploy.ServerDeploymentGroup(stack, 'CodeDeployGroup', {
     failedDeployment: false,
     deploymentInAlarm: false,
   },
+  terminationHook: true,
 });
 
-app.synth();
+new IntegTest(app, 'integ-cloudfront-custom-s3', {
```

Comment:
I might miss something but the test suite name seems weird to me.

Reply:
I'm sorry...　I've updated test name.